dspace-spring-rest/src/main/java/org/dspace/app/rest/BitstreamContentRestController.java

```diff
@@ -26,46 +26,48 @@
 import org.springframework.web.bind.annotation.RestController;
 
 /**
- * This is a specialized controller to provide access to the bitstream binary content
+ * This is a specialized controller to provide access to the bitstream binary
+ * content
  * 
  * @author Andrea Bollini (andrea.bollini at 4science.it)
  *
  */
 @RestController
-@RequestMapping("/api/"+BitstreamRest.CATEGORY +"/"+ BitstreamRest.PLURAL_NAME + "/{uuid:[0-9a-fxA-FX]{8}-[0-9a-fxA-FX]{4}-[0-9a-fxA-FX]{4}-[0-9a-fxA-FX]{4}-[0-9a-fxA-FX]{12}}/content")
+@RequestMapping("/api/" + BitstreamRest.CATEGORY + "/" + BitstreamRest.PLURAL_NAME
+		+ "/{uuid:[0-9a-fxA-FX]{8}-[0-9a-fxA-FX]{4}-[0-9a-fxA-FX]{4}-[0-9a-fxA-FX]{4}-[0-9a-fxA-FX]{12}}/content")
```

Comment:
Please add a comment explaining this very large regex. It is not self-explanatory.